deploy/ControllerInit.sol

```diff
@@ -150,6 +223,15 @@ library ForeignControllerInit {
 
         // Step 2: Configure all rate limits for controller
 
+        // Sanity check all rate limit data
+        _checkUnlimitedData(data.usdcDepositData,  "usdcDepositData");
```

Comment:
Same here. Should be consilidated into one function to make the code more succinct.

Reply:
UPdated